packages/core/src/domain/telemetry/telemetryEvent.types.ts

```diff
@@ -285,6 +285,14 @@ export type TelemetryConfigurationEvent = CommonTelemetryProperties & {
         | 'FragmentViewTrackingStrategy'
         | 'MixedViewTrackingStrategy'
         | 'NavigationViewTrackingStrategy'
+      /**
+       * Whether SwiftUI view instrumentation is enabled
+       */
+      swiftui_view_tracking_enabled?: boolean
+      /**
+       * Whether SwiftUI action instrumentation is enabled
+       */
+      swiftui_action_tracking_enabled?: boolean
```

Comment:
This is coming from the `rum-events-format` sync. Not related to the changes at hand.